### PR TITLE
Feature/upgrade graph api

### PR DESCRIPTION
**Pull request by gaganawhad**
Updates the code to work with v3.0 of FB graph API

### CODE REVIEW

File: app/wrappers/wakes/facebook_metrics_wrapper.rb [hunk 1]
```diff
@@ -19,7 +19,7 @@ def share_counts
     {}.tap do |hash|
       individual_responses.map do |r|
         begin
-          hash[r['id']] = r.fetch('share').fetch('share_count')
+          hash[r['id']] = r.fetch('og_object').fetch('engagement').fetch('count')
```
Comment (benhutton) [inline app/wrappers/wakes/facebook_metrics_wrapper.rb]: @gaganawhad is `r` a hash? Can we use `dig` here?

File: app/wrappers/wakes/facebook_metrics_wrapper.rb [hunk 2]
```diff
@@ -19,7 +19,7 @@ def share_counts
...
         rescue KeyError
           # Some urls can have no 'share' keys. This is generally in the case when the url isn't valid.
```
Comment (benhutton) [inline app/wrappers/wakes/facebook_metrics_wrapper.rb]: this comment should be updated

File: app/wrappers/wakes/facebook_metrics_wrapper.rb [hunk 3]
```diff
@@ -68,7 +68,7 @@ def make_request
   def batched_url_requests
     [].tap do |array|
       urls.each do |url|
-        array << {'method' => 'GET', 'relative_url' => "?id=#{url}"}
+        array << {'method' => 'GET', 'relative_url' => "?id=#{url}&fields=og_object{engagement{count}}"}
```
Comment (benhutton) [inline app/wrappers/wakes/facebook_metrics_wrapper.rb]: @gaganawhad can you link me to the docs for this?

File: app/wrappers/wakes/facebook_metrics_wrapper.rb (hunk 1 above)
Comment (gaganawhad) [reply app/wrappers/wakes/facebook_metrics_wrapper.rb]: @benhutton Unfortunately`dig` doesn't give us `KeyError`.

**Comment by gaganawhad** [comment]
@benhutton I made comments and and update here. Let me know!

File: app/wrappers/wakes/facebook_metrics_wrapper.rb (hunk 1 above)
Comment (benhutton) [reply app/wrappers/wakes/facebook_metrics_wrapper.rb]: @gaganawhad why not try this?

```
hash[r['id']] = r.dig('og_object', 'engagement', 'count')
if hash[r['id']].nil?
  Rails.logger.warn "Share count not received for #{r['id']}. Response is: #{r}"
end
```

**Pull request merged**